PendingReleaseNotes

```diff
@@ -11,6 +11,9 @@
 
 >=16.0.0
 --------
+* volumes/nfs: The ``cephfs`` cluster type has been removed from the
+  ``nfs cluster create`` subcommand.
+
```

Comment:
Add a line saying both ``cephfs`` and ``rgw`` is supported by ``nfs cluster create`` subcommand.

Reply:
added verbiage similar to this to the PendingReleaseNotes.